bodhi/server/validators.py

```diff
@@ -191,15 +191,18 @@ def cache_nvrs(request, build):
 
 
 @postschema_validator
-def validate_nvrs(request, **kwargs):
+def validate_build_nvrs(request, **kwargs):
     """
     Ensure that the given builds reference valid Build objects.
 
     Args:
-        request (pyramid.util.Request): The current request.
+        request (pyramid.request.Request): The current request.
         kwargs (dict): The kwargs of the related service definition. Unused.
     """
-    for build in request.validated.get('builds', []):
+    builds = request.validated.get('builds')
```

Comment:
I'm trying to remember why I changed this in the beginning... Perhaps I should remove it because I don't know what the purpose of this change was anymore.

Reply:
I've looked into it and will remove it.